v0/destinations/posthog/data/PHIdentifyConfig.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+      "destKey": "distinct_id",
+      "sourceKeys": [
+        "userId",
+        "context.traits.userId",
+        "context.traits.id",
+        "anonymousId"
+      ],
+      "required": true
+  },    
+  {
+      "destKey": "timestamp",
+      "sourceKeys": ["originalTimestamp", "timestamp"],
+      "required": true
+    },
+  { "destKey": "context", "sourceKeys": "context", "required": true },
+  { "destKey": "messageId", "sourceKeys": "messageId", "required": true },
+  { "destKey": "$set", "sourceKeys": "context.traits", "required": true }
```

Comment:
source key should be ["traits",  "context.traits"] (refer generic mapping file)

Reply:
Fixed